src/gui/accountstate.h

```diff
@@ -19,6 +19,7 @@
 #include <QByteArray>
 #include <QElapsedTimer>
 #include <QPointer>
+#include <QtWebSockets/QWebSocket>
```

Comment:
You shouldn't need to include from the header, a forward declare would be enough if you had just a pointer (which I advise). Also when you include please use the `#include <ClassName>` style like for the other Qt classes.

Reply:
This can go away now.